core/templates/search/search_advanced.html

```diff
@@ -61,11 +61,27 @@
             <!-- Locations and Dates -->
             <div class="row">
                 <div class="col-md-6">
-                    {{ form.city }}
-                    {{ form.county }}
+                    <div class="col-md-2">
```

Comment:
@karindalziel is there a more bootstrappy way to do space the labels and form elements?  `form-control` or `form-group` or something like that?